controller/array_action/ds8k_rest_client.py

```diff
@@ -143,13 +143,13 @@ def get_used_lun_numbers_by_host(self, host_name):
         mappings = self._client.get_mappings_by_host(host_name)
         return [mapping.id for mapping in mappings]
 
-    def create_volume(self, name, capacity_in_bytes, pool_id, tp):
+    def create_volume(self, name, capacity_in_bytes, pool_id, thin_provisioning):
```

Comment:
even if tp is `thin_provisioning` I don't think it a good name for the parameter.
tp can be every space efficiency we are supporting - including `ese` that we call thin, so it's not the "cross storage" `thin`..

Reply:
we would still need to use tp eventually where?

Reply:
we are using `tp` there

Reply:
exactly.
so in the function call "flow", the value is initially called "space efficiency", but would eventually have to be called `tp`.
we can either call it "space efficiency" all the way until the last call (shown in the link above), or have this intermediate name to explain why "space efficiency" becomes "tp"

Reply:
but why using `thin_provisioning` and not stay with `tp`?

Reply:
because `tp` is too short, it's not descriptive enough (pylint default minimum length is 3)

Reply:
ok, fair enough